Rubberduck.VBEEditor/Events/VBENativeServices.cs

```diff
@@ -337,7 +278,7 @@ private static WindowType ToWindowType(this IntPtr hwnd)
 
         private static string ToClassName(this IntPtr hwnd)
         {
-            var name = new StringBuilder(128);
+            var name = new StringBuilder(255);
```

Comment:
Can we avoid magical number? Surely we defined the max length for the class name somewhere?

Reply:
Yep. I'll fix that. IIR it's a constant in the Windows headers somewhere.